src/test/java/net/openhft/chronicle/core/CoreTestCommon.java

```diff
@@ -1,25 +1,19 @@
 package net.openhft.chronicle.core;
 
+import net.openhft.chronicle.core.internal.JvmExceptionTracker;
 import net.openhft.chronicle.core.io.AbstractReferenceCounted;
-import net.openhft.chronicle.core.onoes.ExceptionKey;
-import net.openhft.chronicle.core.onoes.Slf4jExceptionHandler;
 import net.openhft.chronicle.core.threads.CleaningThread;
 import net.openhft.chronicle.core.threads.ThreadDump;
+import net.openhft.chronicle.testframework.internal.ExceptionTracker;
```

Comment:
Later, the interface needs to be in the public API and not in `internal`.

Reply:
Yeah I agree, but I was thinking we might "incubate" it for a release or so first, especially if we're deprecating or changing ExceptionKey and how all that's done.
Given it's not a feature we anticipate end users making a lot of use of I think it's safe to make it internal until we're happy it's settled?

Reply:
Just as we have an `internal` package we could have an `incubator` package. This is how Java itself is dealing with things like this: https://openjdk.java.net/jeps/11